### PR TITLE
feat(curriculum): adding review JS content to prep exam review page 

### PR DESCRIPTION
**Pull request by jdwilkin4**
There is a long content in the prep exam review page but all of it was copied over from all of the approved individual review pages. 

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/review-recursion/6723d1f0568292cd394d6fb6.md
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/review-recursion/6723d1f0568292cd394d6fb6.md:1]: is this file accidentally added or is part of the PR?

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 1]
```diff
@@ -7,9 +7,2856 @@ dashedName: review-javascript
...
+```
+
+- Apart from `let`, you can also use `const` to declare a variable. However, a `const` variable cannot be reassigned a new value.
+
+```js
+const cityName = 'New York';
+cityName = 'Los Angeles'; // TypeError: Assignment to constant variable.
+```
+
+- Variables declared using `const` find uses in declaring constants, that are not allowed to change through out the code, such as `PI` or `MAX_SIZE`.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:84]:
```suggestion
- Variables declared using `const` find uses in declaring constants, that are not allowed to change throughout the code, such as `PI` or `MAX_SIZE`.
```

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 2]
```diff
@@ -7,9 +7,2856 @@ dashedName: review-javascript
...
+
+- If you need to add or append to an existing string, then you can use the  `+=` operator. This is helpful when you want to build upon a string by adding more text to it over time.
+
+```js
+let message = 'Welcome to programming, ';
+message += 'Asad!';
+console.log(message); // Welcome to programming, Asad!
+```
+
+- Another way you can concatenate strings is to use the concat method. This method joins two or more strings together.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:130]:
```suggestion
- Another way you can concatenate strings is to use the `concat()` method. This method joins two or more strings together.
```

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 3]
```diff
@@ -7,9 +7,2856 @@ dashedName: review-javascript
...
+```
+
+- **Strict Equality (`===`) Operator**: This operator does not perform type coercion and checks if both the types and values are equal.
+
+```js
+console.log(5 === '5'); // false
+```
+
+- **Inequality (`!=`) Operator**: This operator uses type coercion before checking if the values are not equal.
+- **Strict  Inequality (`!==`) Operator**: This operator does not perform type coercion and checks if both the types and values are not equal.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:391]:
```suggestion
- **Strict Inequality (`!==`) Operator**: This operator does not perform type coercion and checks if both the types and values are not equal.
```

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 4]
```diff
@@ -7,9 +7,2856 @@ dashedName: review-javascript
...
+const userSettings = {
+ theme: null,
+ volume: 0,
+ notifications: false,
+};
+
+let theme = userSettings.theme ?? 'light';
+console.log(theme); // light
+```
+
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:482]:
```suggestion
```

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 5]
```diff
@@ -7,9 +7,2856 @@ dashedName: review-javascript
...
+
+## The `Math` Object 
+
+- **The `Math.random()` Method**: This method generates a random floating-point number between 0 (inclusive) and 1 (exclusive). This means the possible output can be 0, but it will never actually reach 1. 
+- **The `Math.max()` Method**: This method takes a set of numbers and return the maximum value.
+- **The `Math.min()` Method**: This method takes a set of numbers and return the minimum value.
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:488]:
```suggestion
- **The `Math.max()` Method**: This method takes a set of numbers and returns the maximum value.
- **The `Math.min()` Method**: This method takes a set of numbers and returns the minimum value.
```

**Review by zairahira — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md [hunk 6]
```diff
@@ -7,9 +7,2855 @@ dashedName: review-javascript
...
+
+- **Logical AND (`&&`) Operator**: This operator checks if both operands are `true`. If both are `true`, then it will return the second value. If either operand is `falsy`, then it will return the `falsy` value. If both operands are `falsy`, it will return the first `falsy` value. 
+
+```js
+const result = true && 'hello';
+
+console.log(result); // hello
+```
+
+- **Logical OR (`||`) Operator**: This operator checks if at least one of the operands is truthy.
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/review-javascript/6723d3cfdd0717d3f1bf27e3.md:469]:
```suggestion
- **Logical OR (`||`) Operator**: This operator checks if at least one of the operands is truthy. If the first value is truthy, then it is returned. If the first value is falsy, then the second value is returned.
```

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**